operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/KubernetesDependentResource.java

```diff
@@ -49,12 +51,17 @@ public KubernetesDependentResource(Class<R> resourceType) {
 
   @Override
   public void configureWith(KubernetesDependentResourceConfig config) {
-    configureWith(config.labelSelector(), config.namespaces(), !config.wereNamespacesConfigured());
+    this.kubernetesDependentResourceConfig = config;
```

Comment:
Shouldn't we call `configureWith(…)` here?

Reply:
we dont, have the context here, so we don't know the controller namespaces. So it cannot be done at this time.

Reply:
So when is it done, then? The expectation with the `configureWith` methods is that once they're called, the dependent resource is set up. In this case, this isn't true anymore and someone calling this method would need to call one of the other versions at some later time…

Reply:
see comment below